LoopKitUI/SupportUI.swift

```diff
@@ -28,4 +30,45 @@ public protocol SupportUI {
     ///   - urlHandler: A handler to open any URLs.
     /// - Returns: A view that will be used in a support menu for providing user support.
     func supportMenuItem(supportInfoProvider: SupportInfoProvider, urlHandler: @escaping (URL) -> Void) -> AnyView?
+
+    ///
+    /// Check whether the given app version for the given `bundleIdentifier` needs an update.  Services should return their last result, if known.
+    ///
+    /// - Parameters:
+    ///    - bundleIdentifier: The host app's `bundleIdentifier` (a.k.a. `CFBundleIdentifier`) string.
+    ///    - currentVersion: The host app's current version (i.e. `CFBundleVersion`).
+    ///    - completion: The completion function to call with any success result (or `nil` if not known) or failure.
+    func checkVersion(bundleIdentifier: String, currentVersion: String, completion: @escaping (Result<VersionUpdate?, Error>) -> Void)
+
+    /// Hand this support an `AlertIssuer` so it may be able to issue alerts.
+    func setAlertIssuer(alertIssuer: AlertIssuer?)
```

Comment:
This seems non-standard. The pump and CGM managers use a delegate as the alert issuer plus they are `AlertResponder`s.